src/main/scala/org/broadinstitute/dsde/firecloud/EntityClient.scala

```diff
@@ -423,4 +427,71 @@ class EntityClient (requestContext: RequestContext, modelSchema: ModelSchema, go
     }
   }
 
+  def pfbImportStatus(workspaceNamespace: String, workspaceName: String, jobId: String, userInfo: UserInfo): Future[PerRequestMessage] = {
+
+    val bucketName = FireCloudConfig.Arrow.bucketName
+
+    def readStatusFile(filename: String, default: String): String = {
+      Try(googleServicesDAO.getObjectContentsAsRawlsSA(bucketName, s"$jobId/$filename")) match {
+        case Success(msg) =>
+          if (msg.trim.nonEmpty) msg else default
+        case Failure(ex) =>
+          logger.warn(s"error reading GCS object gs://$bucketName/$jobId/$filename", ex)
+          default
```

Comment:
if you call this trying to read a `success.json` that exists but the read fails for some reason, is it correct to still return the default here, which is a message saying the import completed successfully?

Reply:
discussed in person - I think it's ok to say success since the success.json read operation and import operation are separate